.github/workflows/test.yaml

```diff
@@ -22,9 +22,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        # TODO(srawlins): Re-enable stable when stable works with analyzer 6.9.0
-        # (Dart 3.6.0).
-        sdk: [dev]
+        sdk: [stable, dev]
```

Comment:
Instead of `stable` might could wire this the oldest supported SDK rev?

Reply:
Hmm, maybe. I guess that would be 3.5.0 right now. Is this the common practice? Oldest stable + dev channels?

Reply:
That's what I've been doing for a while, with the rest of the ecosystem folks @devoncarew to comment?

Reply:
Done